nhaystack-rt/src/nhaystack/server/TagManager.java

```diff
@@ -1266,10 +1274,10 @@ private static int getTrendRecordKind(Type trendRecType)
     /** Every single tag which the server may have auto-generated.  */
     static final String[] AUTO_GEN_TAGS = {
         "axAnnotated",
-        "axHistoryId",
+        "n4HistoryId",
         "axHistoryRef",
         "axPointRef",
-        "axSlotPath", 
+        "n4SlotPath", 
```

Comment:
Breaking change.

Reply:
Re-instated.

Reply:
It's stil lshowing as the axSlotPath has been removed, am I reading it wrong?